contrib/nitrpg/src/test_helper.nit

```diff
@@ -51,4 +51,23 @@ abstract class NitrpgTestHelper
 		game.db_name = db.name
 		return game
 	end
+
+	# Stack of db used for testing.
+	var test_dbs = new Array[MongoDb]
+
+	# Gen a test db with a random name (to avoid race conditions).
+	fun gen_test_db: MongoDb do
+		var db_name = "{get_time}_{1000.rand}"
```

Comment:
I suggest adding some prefix like "nitrpg_test_" to help the database administrator